tests/io/datasets/test_seq.py

```diff
@@ -5,44 +5,49 @@
 import pytest
 
 from libertem.executor.inline import InlineJobExecutor
-from libertem.io.dataset.seq import SEQDataSet
+from libertem.io.dataset.seq import SEQDataSet, _load_xml_from_string, xml_defect_data_extractor
+from libertem.io.dataset.seq import xml_map_sizes, xml_unbinned_map_maker, xml_map_index_selector
+from libertem.io.dataset.seq import xml_defect_coord_extractor, array_cropping
 from libertem.common import Shape
 from libertem.common.buffers import reshaped_view
 from libertem.udf.sumsigudf import SumSigUDF
 from libertem.udf.raw import PickUDF
 from libertem.io.dataset.base import TilingScheme, BufferedBackend, MMapBackend
 
 from utils import get_testdata_path, ValidationUDF
+import defusedxml.ElementTree as ET
```

Comment:
`defusedxml` is a third-party package, so it needs to be added to `setup.py` as a dependency if we want to start using it. Right now, it is probably installed because one of our other dependencies also depends on it, but that is just sheer luck, and can change in the future.